client/jetpack-cloud/components/sidebar/index.tsx

```diff
@@ -115,7 +115,6 @@ const JetpackCloudSidebar = ( {
 				<ul>
 					{ ! isJetpackManage && jetpackAdminUrl && (
 						<SidebarNavigatorMenuItem
-							isExternalLink
```

Comment:
As you know, this would also remove the icon alongside stopping it from opening in a new tab, but just wanted to check if that's definitely intentional! 

There was some pretty extensive discussion about this a while back: https://github.com/Automattic/wp-calypso/pull/33500#issuecomment-498229671.

> The icon simply means you are leaving an experience and entering a new one. It is meant to set an expectation that when you click on this item, you may be going to a different "place". The mechanism by which this happens doesn't matter as much. It could be a URL change, a new tab, a new window, or none of the above.

Of course, that was a while ago so the precedent might have changed! But just thought it was worth confirming whether it's definitely intentional not to preserve the external icon even if it doesn't open in a new tab.

Reply:
Similar comment about retaining the external icon from @nuriapenya in p1708415505170529/1708397914.488109-slack-C06DN6QQVAQ:

> It seems we want to clearly state the two areas apart, so I would try to be as clear as possible that we are bringing the users somewhere else (even thou it’s in the same tab):

Reply:
Thanks y'all! We ended up adding `openInSameTab` to keep the "external link" icon but open in the same tab.